DB/src/main/java/io/deephaven/db/v2/parquet/ParquetReaderUtil.java

```diff
@@ -293,7 +296,10 @@ public static void readParquetSchema(
                 colName = mappedName;
             } else if (parquetColumnName.contains(" ")) {
```

Comment:
Use `DbNameValidator` for this.

Reply:
You can resolve this and make a new issue for it.

Reply:
https://github.com/deephaven/deephaven-core/issues/787